resources/qml/ActionPanel/PrintJobInformation.qml

```diff
@@ -36,30 +36,62 @@ Column
 
         Label
         {
+            id: byLineType
+
             property var printDuration: PrintInformation.currentPrintTime
+            property var columnWidthMultipliers: [ 0.4, 0.3, 0.3 ]
+            property var columnHorizontalAligns: [ TextInput.AlignLeft, TextInput.AlignHCenter, TextInput.AlignHCenter ]
 
-            text:
+            function getMaterialTable()
             {
+                var result = []
+
                 // All the time information for the different features is achieved
                 var printTime = PrintInformation.getFeaturePrintTimes()
                 var totalSeconds = parseInt(printDuration.getDisplayString(UM.DurationFormat.Seconds))
 
                 // A message is created and displayed when the user hover the time label
-                var text = "<table width=\"100%\">"
                 for(var feature in printTime)
                 {
                     if(!printTime[feature].isTotalDurationZero)
                     {
-                        text += "<tr><td>" + feature + ":</td>" +
-                            "<td align=\"right\" valign=\"bottom\">&nbsp;&nbsp;%1</td>".arg(printTime[feature].getDisplayString(UM.DurationFormat.ISO8601).slice(0,-3)) +
-                            "<td align=\"right\" valign=\"bottom\">&nbsp;&nbsp;%1%</td>".arg(Math.round(100 * parseInt(printTime[feature].getDisplayString(UM.DurationFormat.Seconds)) / totalSeconds)) +
-                            "</tr>"
+                        var row = []
+                        row.push(feature + ": ")
+                        row.push("%1".arg(printTime[feature].getDisplayString(UM.DurationFormat.ISO8601).slice(0,-3)))
+                        row.push("%1%".arg(Math.round(100 * parseInt(printTime[feature].getDisplayString(UM.DurationFormat.Seconds)) / totalSeconds)))
+                        result.push(row)
+                    }
+                }
+
+                return result
+            }
+
+            Column
+            {
+                Repeater
+                {
+                    model: byLineType.getMaterialTable()
+                    Row
+                    {
+                        Repeater
+                        {
+                            model: modelData
+                            Text
```

Comment:
Use ```Label``` instead of ```Text``` (https://github.com/Ultimaker/Cura/wiki/QML-Best-Practices). Also since this is using QtQuickControls 2, you need to add ```renderType: Text.NativeRendering```

Reply:
Fixed

Reply:
Thanks.
I keep discovering new Best Practices documents in all kinds of different places :-)